src/api/TourneyRent/TourneyRent.DataLayer/TourneyRent.DataLayer.csproj

```diff
@@ -6,10 +6,6 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="Migrations\" />
-  </ItemGroup>
-
```

Comment:
Does this remove the folder? Revert this change if possible

Reply:
![image](https://user-images.githubusercontent.com/76095383/221009751-c58b45e4-264f-4d48-96e7-19776f87a6ad.png)
I don't see it in my code

Reply:
Red color means that it was removed by you, so it is expected that you can't see it. I guess it's fine, I'll just add it later again if that folder disappears.